README.md

```diff
@@ -6,104 +6,96 @@
 [![Coverage Status](https://coveralls.io/repos/bigcommerce/bigcommerce-api-ruby/badge.png?branch=master)](https://coveralls.io/r/bigcommerce/bigcommerce-api-ruby?branch=master)
 
-This library provides a wrapper around the Bigcommerce REST API for use within
-Ruby apps or via the console.
+This is the official Bigcommerce API client to support our Rest API. You can find more information about becoming a Bigcommerce developer here: [developer.bigcommerce.com](http://developer.bigcommerce.com).
 
-### Note
+#### ⚠️ A note about the current client: ⚠️
 
-If you find anything that is missing or needs clean up, please feel free to fork
-it and submit a changes with your pull request.
+We have introduced a new major version of the API client and it is a complete rewrite (for the better). If you want to see the old version of the API client, please view it here: [Bigcommerce API client v0.x](https://github.com/bigcommerce/bigcommerce-api-ruby/tree/0.x). We recommend that developers upgrade to the latest client, but we will still support our developers who are unable to upgrade.
 
-## Requirements
 
-- Ruby 1.9+
+## Installation
 
-To connect to the API, you need the following credentials:
+Bigcommerce is available to be pulled down from Rubygems:
 
-For the OAuth API:
+```sh
+gem install bigcommerce
+```
 
-- Create and register an app at [developer.bigcommerce.com](https://developer.bigcommerce.com).
-- Have a callback URL defined
-- Prepare the `client_id` credential
+You can also add it to your Gemfile.
 
-For the Legacy API:
+```rb
+gem 'bigcommerce'
+```
 
-- Secure URL pointing to a Bigcommerce store
-- Username of an authorized admin user of the store
-- API key for the user
+## Getting Started
 
-A valid API key is required to authenticate requests. To grant API access for a user, go to Control Panel > Users > Edit User and make sure that the 'Enable API access?' checkbox is ticked.
+In order to make requests to our API, you must have registered as a developer and have your credentials ready.
 
-## Installation
+We currently have two different types of authentication schemes you can use depending on your use-case.
 
-Download the lib folder and copy it to a path accessible within your app, or
-install the package directly from Rubygems:
+#### Private Apps
 
-```sh
-gem install bigcommerce
-```
+To develop a custom integration for one store, your app needs to use Basic Authentication.
 
-## Configuration
+[More Information](https://developer.bigcommerce.com/api/legacy/basic-auth)
 
-#### OAuth Authentication
+#### Single Click Apps
 
-In order to create a new Bigcommerce application, please visit [developer.bigcommerce.com](https://developer.bigcommerce.com). Its free to sign up and only takes a moment. Once you have created your app, you can get a `client_id`.
+Public apps (also known as Single-Click Apps) must use OAuth authentication. This will allow you to write apps which other businesses can install into their stores.
```

Comment:
"Public apps can be submitted to Bigcommerce App Store, allowing other businesses to install it in their Bigcommerce stores."